po/pl.po

```diff
@@ -1120,9 +1151,8 @@ msgid "Sodium"
 msgstr "Sód"
 
 #: ../libelemental/data.cc:54
-#, fuzzy
 msgid "English: soda; Latin: natrium (sodium carbonate)"
-msgstr "Łacina: nuritum (soda gazowana)"
+msgstr "Łacińskie: natrium (natryt - skała)"
```

Comment:
This corrects the previous Latin etymology but (like the old translation) omits the English etymology. Putting it back in: "Angielskie: soda; łacińskie: natrium (natryt - skała)". (Note switch to lowercase ł because of the changed punctuation.)